projeckt/1.html

```diff
@@ -0,0 +1,44 @@
+<!Doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>table</title>
+</head>
+<body background="1.jpg">
+  <table  width="100%">
+    <tr>
+    <td colspan="2" align="center" text="red"><h1>Cтатистика турнира
+    </td>
+    <tr>
+    <td>
+      <a href="1.html"  align="left">Cтатистика турнира</a>
+      <br>
+      <br>
+      <a href="2.html"  align="left">Интервью с главным судьей</a>
+      <br>
+      <br>
+      <a href="3.html"  align="left">Конкурсные вопросы</a>
+    </td>
+    <td>
+      <ul>
```

Comment:
uncorrect list from 23 line to 37

under <ul> we can set only <li> tags

Uncorrect: <ul><ul></ul></ul>

Correct <ul>
                    <li><ul></ul>
                    </li>
            </ul>